core/object/message_queue.cpp

```diff
@@ -93,7 +96,7 @@ Error CallQueue::push_callablep(const Callable &p_callable, const Variant **p_ar
 
 	if ((page_bytes[pages_used - 1] + room_needed) > uint32_t(PAGE_SIZE_BYTES)) {
 		if (pages_used == max_pages) {
-			ERR_PRINT("Failed method: " + p_callable + ". Message queue out of memory. " + error_text);
+			std::cout << (wchar_t)("Failed method: " + p_callable + ". Message queue out of memory. " + error_text).get_data();
```

Comment:
This isn't correct, you are casting to a value, not a pointer, and it isn't `wchar_t`, it's `char32_t`, also this doesn't end the line